packages/styles/forms.css

```
@@ -118,6 +137,10 @@ textarea.Field--has-error,
   color: var(--field-border-color-error);
 }
 
+.Icon--checkbox-unchecked.Checkbox__overlay:active.Field--has-error:not(.Checkbox__overlay--disabled) {
```

```suggestion
.Checkbox__overlay:active.Field--has-error:not(.Checkbox__overlay--disabled) {
```

This feels like an overly specific selector, can we not just use `.Checkbox__overlay`?

---

Ah in this case, @bobbyomari mentioned to me that it should only show the red active fill state when checking the unchecked box, not when unchecking the checked box

---

If that is the case, we should be checking the checked state and not the class.

---

Something like `.Checkbox__overlay:not(:checked):active...`